saleor/core/emails.py

```diff
@@ -14,3 +19,34 @@ def get_email_context():
         "site_name": site.name,
     }
     return send_email_kwargs, email_template_context
+
+
+def send_set_password_email_with_url(redirect_url, user, staff=False):
```

Comment:
Shouldn't belong to the `account` module?

Reply:
Yes, I will move that.